databases/week1/class/homework.sql

```diff
@@ -0,0 +1,83 @@
+-- Find out how many tasks are in the task table
+SELECT 
+  COUNT(title) 
+FROM 
+  task;
+  
+-- Find out how many tasks in the task table do not have a valid due date
+SELECT 
+  COUNT(title) 
+FROM 
+  task 
+WHERE 
+  due_date IS NULL;
+
+-- Find all the tasks that are marked as done
+SELECT 
+  * 
+FROM 
+  task 
+  JOIN status ON task.status_id = status.id 
+WHERE 
+  status.name LIKE "%done%";
```

Comment:
status.name != 'done';
_Pointer_
Use LIKE operator for only pattern matches

Reply:
Thanks, now it's clear for me! I wasn't sure about it 'cause equality operator didn't work in some cases (now, I assume this is as you've mentioned "pattern cases").